### PR TITLE
Add unit tests for `$addLabel()` action

### PR DESCRIPTION
**Pull request by shay2025**
This pull request implicates the following:
- add unit tests for `$addLabel()` action;
- export mocks file to an independent package and adapt the existing unit tests to that change (for now, only TestCommentOnce_XXX funcs);
- remove `$addLabel()` args len check since, in the type declaration, we already declare that this built-in must have one argument;
- add `FailOnError` function that fails and prints a message if an error occurs.

### CODE REVIEW

**Comment by shay2025** [comment]
The `FailOnError` function was removed since it was thought that it was not needed.
It was also made the reestructure of the actions builtins.

File: plugins/aladino/actions/addLabel_test.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	}
+
+	args := []aladino.Value{aladino.BuildStringValue(label)}
+	err = addLabel(mockedEnv, args)
+
+	assert.Equal(t, err.(*github.ErrorResponse).Message, failMessage)
+}
+
+func TestAddLabel(t *testing.T) {
+	labelA := "bug"
```
Comment (ferreiratiago) [inline plugins/aladino/actions/addLabel_test.go:92]: Suggestion: `label`.

**Comment by shay2025** [comment]
@marcelosousa @ferreiratiago the only files that introduce new code are:
- `mocks/aladino/mocks.go`: package name change and capitalize function names for export purposes.
- `plugins/aladino/actions/addLabel.go` : [Remove addLabel args len check](https://github.com/reviewpad/reviewpad/pull/32/commits/93ada2a17b8400fcf938e800c9c093c76222f4d4)
- `plugins/aladino/actions/addLabel_test.go`: unit tests for `$addLabel` action builtin - main feature
- `plugins/aladino/builtins.go`: get builtins code
- `plugins/aladino/actions/assignTeamReviewer_test.go`: added a way to go get the action builtin code by accessing the builtins structure 
https://github.com/reviewpad/reviewpad/blob/3f095bc37387f4346941a855f73dc7e8ab16430c/plugins/aladino/actions/assignTeamReviewer_test.go#L21
- `plugins/aladino/actions/commentOnce_test.go`: added a way to go get the action builtin code by accessing the builtins structure
https://github.com/reviewpad/reviewpad/blob/3f095bc37387f4346941a855f73dc7e8ab16430c/plugins/aladino/actions/commentOnce_test.go#L25

**Review by ferreiratiago — APPROVED** [review]
Some minor improvemts.

File: plugins/aladino/actions/addLabel_test.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	if err != nil {
+		log.Fatalf("mockDefaultEnv failed: %v", err)
+	}
+
+	args := []aladino.Value{aladino.BuildStringValue(label)}
+	err = addLabel(mockedEnv, args)
+
+	assert.Nil(t, err)
+	assert.Equal(t, wantLabels, gotLabels)
+}
```
Comment (marcelosousa) [inline plugins/aladino/actions/addLabel_test.go:123]: Please add a new line here.

**Review by marcelosousa — APPROVED** [review]
Just a small lint issue.

**Pull request merged**